public/data/ankanbag101.json

```
@@ -0,0 +1,67 @@
+{
+    "name": "Ankan Bag",
+    "type": "personal",
+    "bio": "An alien 👽 trying to make a difference in the tech world 🔥",
+    "avatar": "https://github.com/ankanbag101.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/ankanbag101",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AnkanBag06",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://linkedin.com/in/ankan-bag-74a238222/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Email me",
+        "url": "mailto:bagankan@gmail.com",
+        "icon": "envelope"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started GitHub account",
```

I am not sure if you **Start** github accounts

```suggestion
        "title": "Created GitHub account",
```

---

Yes, started works just fine here. No need to change.